specification/under_development/Base-Profile.md

```diff
@@ -0,0 +1,160 @@
+# Base Profile
+
+This profile defines a subset of the QIR specification to support a coherent set of functionalities and capabilities that might be offered by a system. The Base Profile specifies the minimal requirements for executing quantum programs. Execution of a Base Profile compliant program does, for instance, not require support executing quantum instructions dependent on measurement outcomes. The only requirements for a backend are to be able to perform unitary transformations on the quantum state, as well as measurements at the end of the program.
+
+TODO: table with list of features that need to be supported by hardware, and what it doesn't need to support?  
```

Comment:
I found this information helpful, though maybe not precisely what we intend to document in this location: https://github.com/microsoft/qsharp-language/blob/ageller/profile/Specifications/QIR/Base-Profile.md#quantum-instruction-set

Reply:
Not in this file, but I intend to create a separate file documenting used instructions (each of which may or may not be supported by a specific backend). The profile specification on the other hand also contains a table listing *all* llvm instructions that can occur in the program further below.